Source/ACE/Network/GameAction/QueuedGameAction.cs

```diff
@@ -21,10 +24,19 @@ public QueuedGameAction(uint objectId, uint secondaryObjectId, GameActionType ac
             this.ActionType = actionType;
         }
 
+        public QueuedGameAction(uint objectId, GeneralMotion motion, GameActionType actionType)
+        {
+            this.ObjectId = objectId;
+            this.Motion = motion;
+            this.ActionType = actionType;
+        }
+
         public uint ObjectId { get; private set; }
 
         public uint SecondaryObjectId { get; private set; }
 
         public GameActionType ActionType { get; private set; }
+
```

Comment:
There's a lot of things we're gonna need queues for, I don't see how this doesn't keep growing :|